yang/vyatta-protocols-frr-zebra-v1.yang

```diff
@@ -0,0 +1,227 @@
+			}
+			leaf delay {
+				type uint32 {
+  				range "0..16777215";
+				}
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Unidirectional Average Link Delay";
+				description "Unidirectional Average Link Delay";
+			}
+			leaf delay-variation {
+				type uint32 {
+  				range "0..16777215";
+				}
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Unidirectional Link Delay Variation";
+				description "Unidirectional Link Delay Variation";
+			}
+			leaf max-bw {
+				type ieee-float;
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Maximum bandwidth that can be used";
+				description "Maximum bandwidth that can be used";
+			}
+			leaf max-rsv-bw {
+				type ieee-float;
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Maximum bandwidth that may be reserved";
+				description "Maximum bandwidth that may be reserved";
+			}
+			leaf metric {
+				type uint32 {
+  				range "1..4294967295";
+				}
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Link metric for MPLS-TE purpose";
+				description "Link metric for MPLS-TE purpose";
+			}
+			leaf neighbor {
+				type types:ipv4-address;
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Configure remote ASBR information (Neighbor IP address and AS number)";
+				description "Configure remote ASBR information (Neighbor IP address and AS number)";
+			}
+			leaf packet-loss {
+				type float;
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Configure remote ASBR information (Neighbor IP address and AS number)";
+				description "Configure remote ASBR information (Neighbor IP address and AS number)";
+
+			}
+			leaf res-bw {
+				type float;
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Unidirectional Residual Bandwidth";
+				description "Unidirectional Residual Bandwidth";
+			}
+			leaf unrsv-bw {
+				type float;
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Unreserved bandwidth at each priority level";
+				description "Unreserved bandwidth at each priority level";
+			}
+			leaf use-bw {
+				type float;
+				must "(../enable)" {
+					error-message "link-params enable must be set."
+				}
+      	configd:help "Unidirectional Utilised Bandwidth";
+				description "Unidirectional Utilised Bandwidth";
+
+			}
+    }
+  }
+
+  augment /if:interfaces/interfaces-dataplane:dataplane {
+    uses link-params;
+  }
+  augment /if:interfaces/interfaces-dataplane:dataplane/interfaces-dataplane:vif/interfaces-dataplane {
```

Comment:
This path is invalid

Reply:
The path is now correct after adding the :ip at the end of the path